forms/copy.py

```diff
@@ -6,13 +6,15 @@
 
 from models import Copy
 
-""" Regex return True for assets like "wr123456".
+""" Regex return True for assets like "ab123456" or "123456".
```

Comment:
Capitalize those `ab123...` and `123...` so they really match the regex that is used.

Reply:
Regexp accepts both capital and non-capital letters, notice ignorecase flag in regexp ;)

Reply:
If so, then it's just easier to change the regex itself from the current one to this:
`^([A-Za-z]{2}[0-9]{6}|[0-9]{6})$`
The `[A-Za-z]{2}` part tells that the regex accepts excatly two characters from latin alphabet both uppercase and lowercase. No need to pass a parameter to a function. ;)